lidar_topic: /livox/lidar
map_topic: /global_points
debug: False

vins_path: "/media/qzj/Dataset/code/catkin/catkin_roadslam/src/RGBD-Inertial/vins_outputs/results-project_2022-11-18-00-02-32"
ground_truth_path: "/media/qzj/Extreme SSD/datasets/CV_Project/omni_depth/"
bag_path: "/media/qzj/Extreme SSD/datasets/CV_Project/trainning_data/project_2022-11-18-00-02-32.bag"

fisheye_topic: /arducam/image/compressed

fisheye0:
  topic_name: /arducam/image_0/compressed

fisheye1:
  topic_name: /arducam/image_1/compressed

fisheye2:
  topic_name: /arducam/image_2/compressed

fisheye3:
  topic_name: /arducam/image_3/compressed

Ex_10: [ 0.007358773605000313,-0.0017131205279386232,0.9999714564271793,0.08306763551904046,
         -0.00520646545064855,0.9999849124425988,0.0017514578745332853,0.0004568450936354583,
         -0.999959369758866,-0.005219205421500188,0.007349743276239551,-0.0841265793964429,
         0.0, 0.0, 0.0, 1.0 ]
Ex_21: [-0.9999850591540741, 0.00378532264350626, -0.003943703983620389, 0.025160119778843224,
        0.0038575339434209605, 0.9998220337841596, -0.018466732022351738, -0.002464374076230847,
        0.0038730995986702528, -0.018481669085733433, -0.999821697607782, -0.1421829287055014,
        0.0, 0.0, 0.0, 1.0]
Ex_32: [0.00020095914086487587, -0.00971295216421803, -0.9999528079743011, -0.08379186342236422,
        0.0028777982524269847, 0.9999486931080315, -0.009712333847674788, 5.47199151479378e-06,
        0.999995838937669, -0.0028757106610320634, 0.00022890074704001728, -0.08273769306568317,
        0.0, 0.0, 0.0, 1.0]
Ex_c2: [-0.70675836,  0.        ,  0.70745503,0.04999,
        0.        , -1.        ,  0.        ,0.0155,
        0.70745503,  0.        ,  0.70675836,-0.04158,
        0.0, 0.0, 0.0, 1.0]


l515:
  camera_model: pinhole
  distortion_coeffs: [0.10981707696733878, -0.2596508152545224, -0.00047874650512554713, -0.0018447855562127064]
  distortion_model: radtan
  intrinsics: [583.6092372748109, 583.8529457523713, 322.3093788988131, 235.2873500376188]
  resolution: [640, 480]
  depth_topic: /camera/aligned_depth_to_color/image_raw
  # cam to body
  body_T_cam0: [ 9.9991900792232358e-01, -1.1542669966461024e-02, -5.3613772187374106e-03, -1.7882366959740513e-02,
                 1.1312183961713539e-02, 9.9908706668997271e-01, -4.1195481145902754e-02, 4.6900332485262826e-03,
                 5.8319884818635263e-03, 4.1131495752907365e-02, 9.9913672135873177e-01, 3.1353698282641562e-02,
                 0., 0., 0., 1. ]

rot_diff: [0.994378,-0.0838856,-0.0646112,0.0657367,0.0107227,0.997779,-0.0830065,-0.996418,0.0161768]